clients/feeder/dbc2val/j1939reader.py

```diff
@@ -0,0 +1,216 @@
+"""
```

Comment:
This needs a copyright header on top similar too

```

########################################################################
# Copyright (c) 2020 Robert Bosch GmbH
#
# This program and the accompanying materials are made
# available under the terms of the Eclipse Public License 2.0
# which is available at https://www.eclipse.org/legal/epl-2.0/
#
# SPDX-License-Identifier: EPL-2.0
########################################################################
```

(You can write (c) Bosch as the example, or your name, both fine)